GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/UpgradeVerbTests.cs

```diff
@@ -142,12 +142,12 @@ public void ProjFSPreCheck()
                 expectedReturn: ReturnCode.GenericError,
                 expectedOutput: new List<string>
                 {
-                    "ERROR: `gvfs upgrade` is not supported because you have previously installed an out of band ProjFS driver.",
+                    "ERROR: `gvfs upgrade` is only supported after the \"Windows Projected File System\" optional feature",

```

Comment:
It looks like when we're comparing output/errors, we're testing if the expected output is a substring of the actual output (assuming this test passes). 

Before, we appear to be testing the full string. Is it intentional that expectedOutput behaves in this way? If we want this test to have the same behavior that it used to, we could update this string to be the full text from `InstallerPreRunChecker.cs`.